homeassistant/components/google/__init__.py

```diff
@@ -253,7 +251,7 @@ def step2_exchange(now: datetime) -> None:
 def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the Google platform."""
 
-    if not (conf := config.get(DOMAIN, {})):
+    if not (conf := CONFIG_SCHEMA(config.get(DOMAIN, {}))):
```

Comment:
I'm not following this change. What does it achieve?

Reply:
For some reason I had the impression that this was needed for the default values to be set.   I had to do this in `stream` to make it work, but maybe that is only when empty configuration via `default_config` or something.
https://github.com/home-assistant/core/blob/2957f4ce85055e26a5768847e56284ff84b9f543/homeassistant/components/stream/__init__.py#L157

I've verified everything passes when it is removed, and i've removed it.